wicket-core/src/main/java/org/apache/wicket/csp/CSPHeaderConfiguration.java

```diff
@@ -99,7 +100,8 @@ public CSPHeaderConfiguration unsafeInline()
 			.add(CONNECT_SRC, SELF)
 			.add(FONT_SRC, SELF)
 			.add(MANIFEST_SRC, SELF)
-			.add(CHILD_SRC, SELF);
+			.add(CHILD_SRC, SELF)
+			.add(BASE_URI, SELF);
```

Comment:
The new directive should be added to the javadoc of this method as well.

Reply:
Should be addressed :)
Thanks for review!